typing_requirements.txt

```
@@ -0,0 +1,2 @@
+mypy==0.770
```

mypy uses parts of the C-API that PyPy has not implemented, so this should be (not tested,

```suggestion
mypy==0.770; platform_python_implementation != "PyPy"
```